src/ws/cockpit.service.in

```diff
@@ -4,8 +4,8 @@ Documentation=man:cockpit-ws(8)
 Requires=cockpit.socket
 
 [Service]
-ExecStartPre=@sbindir@/remotectl certificate --ensure --user=root --group=@group@ --selinux-type=@selinux_config_type@
+ExecStartPre=@sbindir@/remotectl certificate --ensure --user=@cockpit_user@ --group=@cockpit_group@ --selinux-type=@selinux_config_type@
```

Comment:
These prefix variables are expanding to something like ```${exec_prefix}``` then if I run ```make install``` i end up with those values in my cockpit.service file.